dotcom-rendering/src/components/Discussion/Comments.stories.tsx

```diff
@@ -101,42 +111,43 @@ InitialPage.decorators = [
 	]),
 ];
 
-export const Overrides = () => (
-	<div
-		css={css`
-			width: 100%;
-			max-width: 620px;
-		`}
-	>
-		<Comments
-			shortUrl="p/39f5z"
-			pageSizeOverride={50}
-			orderByOverride="recommendations"
-			baseUrl="https://discussion.theguardian.com/discussion-api"
-			isClosedForComments={false}
-			additionalHeaders={{
-				'D2-X-UID': 'testD2Header',
-				'GU-Client': 'testClientHeader',
-			}}
-			expanded={true}
-			onPermalinkClick={() => {}}
-			onExpand={() => {}}
-			apiKey=""
-			idApiUrl="https://idapi.theguardian.com"
-			page={3}
-			setPage={() => {}}
-		/>
-	</div>
-);
-Overrides.storyName = 'with page size overridden to 50';
-Overrides.decorators = [
-	splitTheme([
-		{
-			...format,
-			theme: Pillar.Opinion,
-		},
-	]),
-];
+// we dont need this now but the logic should be tested in discussion tests
+// export const Overrides = () => (
+// 	<div
+// 		css={css`
+// 			width: 100%;
+// 			max-width: 620px;
+// 		`}
+// 	>
+// 		<Comments
+// 			shortUrl="p/39f5z"
+// 			baseUrl="https://discussion.theguardian.com/discussion-api"
+// 			isClosedForComments={false}
+// 			additionalHeaders={{
+// 				'D2-X-UID': 'testD2Header',
+// 				'GU-Client': 'testClientHeader',
+// 			}}
+// 			expanded={true}
+// 			onPermalinkClick={() => {}}
+// 			onExpand={() => {}}
+// 			apiKey=""
+// 			idApiUrl="https://idapi.theguardian.com"
+// 			page={3}
+// 			setPage={() => {}}
+// 			filters={filters}
+// 			setFilters={() => {}}
+// 		/>
+// 	</div>
+// );
+// Overrides.storyName = 'with page size overridden to 50';
+// Overrides.decorators = [
+// 	splitTheme([
+// 		{
+// 			...format,
+// 			theme: Pillar.Opinion,
+// 		},
+// 	]),
+// ];
```

Comment:
What about simply not exporting this story for now?

Reply:
Yep, this has been moved into the discussion storybook